src/test/java/ca/corefacility/bioinformatics/irida/service/impl/unit/user/UserServiceImplTest.java

```diff
@@ -137,7 +135,7 @@ public void testReadNewAnnouncements() {
 		Announcement second_announcement = new Announcement("test 2", "this is also a test message", false, u);
 		List<Announcement> unreadAnnouncements = Lists.newArrayList(first_announcement, second_announcement);
 
-		when(announcementUserJoinRepository.getAnnouncementsUnreadByUser(any(User.class))).thenReturn(unreadAnnouncements);
```

Comment:
Another one, any reason for checking for `isNull()` in place of `any(User.class)`?

Reply:
Because the call to the function actually passed null.

Mockito 1.x which we were on would accept null values for any() even if a class if expected.